modules/upgrade/pages/proxy-uyuni.adoc

```diff
@@ -80,3 +90,19 @@ spacewalk-proxy start
 
 If you need to update many proxies, you can create an action chain of this command sequence on the {productname} Server.
 You can use the action chain to perform updates on multiple proxies at the same time.
+
+
+
+////
+== Update the Proxy (Y)
+
+To update a proxy use the [guimenu]``Product Migration``:
+
+// FIXME: Update images
```

Comment:
I guess this can be removed?

Reply:
If we really want to enable Product migration we probably must check the screen shots...)